Woof/Woof/View/OwnerFlow/OwnerOnboardingView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+/// A view that onboards the owner and also collects mandatory information about them.
+struct OwnerOnboardingView: View {
+    /// The name of the owner.
+    @Binding var name: String
+
+    /// The surname of the owner.
+    @Binding var surname: String
+
+    /// The phone of the owner.
+    @Binding var phone: String
+
+    /// The owner's city.
+    @Binding var city: String
+
+    /// The address of the owner street and building
+    @Binding var address: String
+
+    /// A boolean indicating whether the save button should be enabled.
+    var isSaveButtonEnabled: Bool {
+        !name.isEmpty && !phone.isEmpty && !city.isEmpty && !address.isEmpty
```

Comment:
All these should be in the view model.
Just create a view model with published properties and use them in view.
WDYT?